### PR TITLE
Added flag signing program

### PR DESCRIPTION
**Pull request by the-c0d3br34k3r**

### CODE REVIEW

File: redblueapp/Dockerfile [hunk 1]
```diff
@@ -3,13 +3,19 @@ FROM drupal:8.5.0
...
+RUN apt-get -y install libssl-dev
 
 ADD supervisord.conf /etc/supervisor/conf.d/supervisord.conf 
 
 COPY drupal-8.5.0 /var/www/html
 RUN mkdir /opt/net-diag
 COPY net-diag /opt/net-diag
 COPY drupal-8.5.0 /var/www/html
+
+ENV MASTER_SALT ABCD
```
Comment (paul-ion) [inline redblueapp/Dockerfile:15]: We can leave this one out. The master salt will be passed in by ECS or by docker with -e

File: redblueapp/drupal-8.5.0/red_ch1.cpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    stringstream ss;
+    for(int i = 0; i < SHA256_DIGEST_LENGTH; i++) {
+        ss << hex << setw(2) << setfill('0') << (int) hash[i];
+    }
+    return ss.str();
+}
+
+int main(int argc, char *argv[]) {
+    char* master_salt = getenv("MASTER_SALT");
+    string challenge_id = "ID1";
```
Comment (paul-ion) [inline redblueapp/drupal-8.5.0/red_ch1.cpp:28]: Let's use "red_ch1" here

File: redblueapp/net-diag/red_ch2.cpp [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    stringstream ss;
+    for(int i = 0; i < SHA256_DIGEST_LENGTH; i++) {
+        ss << hex << setw(2) << setfill('0') << (int) hash[i];
+    }
+    return ss.str();
+}
+
+int main(int argc, char *argv[]) {
+    char* master_salt = getenv("MASTER_SALT");
+    string challenge_id = "ID2";
```
Comment (paul-ion) [inline redblueapp/net-diag/red_ch2.cpp:28]: Let's use "red_ch2" here

File: redblueapp/Dockerfile (hunk 1 above)
Comment (the-c0d3br34k3r) [reply redblueapp/Dockerfile]: Done

File: redblueapp/net-diag/red_ch2.cpp (hunk 3 above)
Comment (the-c0d3br34k3r) [reply redblueapp/net-diag/red_ch2.cpp]: Done

File: redblueapp/drupal-8.5.0/red_ch1.cpp (hunk 2 above)
Comment (the-c0d3br34k3r) [reply redblueapp/drupal-8.5.0/red_ch1.cpp]: Done

**Comment by the-c0d3br34k3r** [comment]
Thank you @paul-ion ! The requested changes have been made.

**Comment by paul-ion** [comment]
Many thanks!

**Pull request merged**